site/docs/docs/05-Counter.md

```diff
@@ -41,19 +41,20 @@ val noLabelsExample = {
       "Example Counter of Failure"
     )
     _ <- IO(println("Action Here")).guaranteeCase{
-      case ExitCase.Completed => successCounter.inc
+      case OutcomeIO.Succeeded(_) => successCounter.inc
```

Comment:
This still isn't compiling. What's the correct way to write this in CE3?

Reply:
`OutcomeIO` is a type alias. The correct way to write it is `Outcome.Succeeded(_)`.